packages/govuk-frontend/src/govuk/components/task-list/task-list.yaml

```diff
@@ -0,0 +1,222 @@
+params:
+  - name: items
+    type: array
+    required: true
+    description: Array of item objects.
+    params:
+      - name: classes
+        type: string
+        required: false
+        description: Classes to add to the item `div`
+      - name: title.text
+        type: string
+        required: true
+        description: If `html` is set, this is not required. Text to use within the title. If `html` is provided, the `text` argument will be ignored.
+      - name: title.html
+        type: string
+        required: true
+        description: If `text` is set, this is not required.
+      - name: title.classes
+        type: string
+        required: false
+        description: Classes to add to the title wrapper
+      - name: tag.text
+        type: string
+        required: true
+        description: If `html` is set, this is not required. Text to use within the value. If `html` is provided, the `text` argument will be ignored.
+      - name: tag.html
```

Comment:
These docs are out of date. Below, you include a `status` attribute where you store either the tag macro data or the flat data for the status `div`. The `status` attribute isn't documented here, and this documentation reflections a previous implementation where the `tag` attribute include the information for rendering the status which used the new tag design.

Reply:
@owenatgov  Thanks - this all needed updating. I've updated it now to show that `tag` is within `status`. It’s a bit tricky to describe which things are required or not, as the tag overrides the status `text` or `html` if present.

Could you take another look and let me know if what I’ve added makes sense?